easybuild/tools/filetools.py

```diff
@@ -367,6 +367,7 @@ def search_file(paths, query, short=False, ignore_dirs=None, silent=False):
             dirnames[:] = [d for d in dirnames if not d in ignore_dirs]
 
         if hits:
+            hits = sorted(hits)
```

Comment:
again, `hits` may be used again deeper down or higher up, who knows

`sorted([])` is pretty cheap anyway ;)